### PR TITLE
UI

### PR DESCRIPTION
**Pull request by thisandagain**
#### Major Changes
- Implements user interface as per [mockups](https://redpen.io/p/tc87e54ffcdc2981c7)
- Removes a large amount of boilerplate and unused libraries
- Introduces a simple in-memory LRU cache for Github requests to help with performance (see performance issues with Waffle.io)

#### Meh
- Still uses server-side templates (HBS)
- Uses LESS on the client-side (not compiled)
- Not tested thoroughly outside of latest Firefox, Chrome, and Mobile Safari

### CODE REVIEW

**Comment by thisandagain** [comment]
@secretrobotron R?

**Comment by thisandagain** [comment]
@secretrobotron R?

File: public/css/styles.less [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+@import "navigation";
+
+@import "views/splash";
+@import "views/add";
+@import "views/sprint";
+@import "views/calendar";
+
+//
+
+// Header
```
Comment (secretrobotron) [inline public/css/styles.less]: Any reason this is left here?

Comment (thisandagain) [inline public/css/styles.less]: Whoops. Nope. Will remove.

**Comment by secretrobotron** [comment]
* Looks good functionally.
* We might want to change some of the links soon, but this is a good start.
* Some visual problems, but nothing worth stalling over.

**Comment by thisandagain** [comment]
Thanks for reviewing. Could you list out a bit more on your last two items?

**Comment by secretrobotron** [comment]
* You're just pointing to github issues (e.g. design and product handbooks)., which is good to indicate progress, but potentially weird for someone looking at the site. Alternative is to just remove them until we have some better content.
* Why do you do ^^ redirects on the server instead of just using real links? Then people can see if they're about to leave the site.
* The Get Involved page is buried within webmaker.org atm. For someone who's just been reading build.webmaker.org, maybe there's a better set of things we can ask people to do.
* The sub-menu looks like it's too far over: screenshot 2015-01-16 12 00 58
* The labels on /now aren't clickable. Should they link to the corresponding github page?
* It would be great for the /add page to show you a link to the github issue it just created instead of navigating there.

**Comment by secretrobotron** [comment]
None of these are blockers though. Can file them as improvements.

**Comment by thisandagain** [comment]
Awesome. Thanks.

- Guides: Wanted to show progress and give folks an indication of where those things are being tracked but I think you are correct in that it's pretty useless without a gitbook to direct to (cc @simonwex ). Will remove.
- Redirects: Easier to say "go to build.w.o/dashboard" than "go to https://mozillafoundation.geckoboard.com/dashboards/F62088172D822E2A"
- Get Involved: +1. I'd love to design / build a more targeted set of communication.
- Sub-Menu: +1 will file bug
- Labels: +1 will file bug
- Add: +1 will file bug

**Comment by simonwex** [comment]
@thisandagain for now you can use this link: http://mozillafoundation.gitbooks.io/mozilla-foundation-engineering-handbook/

I'll likely move the content to a gh-pages branch and cname it.

**Comment by thisandagain** [comment]
@simonwex Awesome. Thanks.

**Pull request merged**